### PR TITLE
fix(react): prepare the next release of react

### LINKED ISSUE #39: react@16.0.0-alpha.6
Support the upcoming version of React https://github.com/callemall/material-ui/issues/6520.

### PR DESCRIPTION
**Pull request by oliviertassinari**
Fix #39
Revert 9d25547f646790e2d5404f87c53b20cb7a2fc44d

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -1,7 +1,5 @@
 [ignore]
 
-.*/node_modules/fbjs/*
```
Comment (RaulTsc) [inline .flowconfig]:
```
Launching Flow server for ...
Spawned flow server (pid=25419)
Logs will go to ...
node_modules/react-event-listener/src/index.js:5
  5: import shallowEqual from 'fbjs/lib/shallowEqual';
                              ^^^^^^^^^^^^^^^^^^^^^^^ fbjs/lib/shallowEqual. Required module not found


Found 1 error
error Command failed with exit code 2.
```

Comment (RaulTsc) [inline .flowconfig]: Any reason why this is removed? It's failing with `"flow-bin": "^0.38.0",` for me.

Comment (RaulTsc) [inline .flowconfig]: I mean I can ignore it myself but I think it will cause 1-2hrs of pain for others.

Comment (oliviertassinari) [inline .flowconfig]: This change only affects the project dev environment. It was removed as flow addressed the issue. Upgrade your version of flow, you are quite behind.

Comment (RaulTsc) [inline .flowconfig]: Yeah, my CI was failing. Upgrading to 0.45.0 now. Thx

**Pull request merged**